gen/builders/predicates.go

```diff
@@ -33,6 +34,17 @@ func ExitCode(expect exitcode.ExitCode) ApplyRetPredicate {
 	}
 }
 
+// MessageReturns returns an ApplyRetPredicate that passes if the message response
+// matches the argument.
+func MessageReturns(expect []byte) ApplyRetPredicate {
```

Comment:
This would be more concise if the arg was a `CBORUnmarshaller`. WDYT? Rationale: if we're comparing against something that has indeed been returned, we know it must be CBOR.